gffquant/profilers/panda_profiler.py

```
@@ -0,0 +1,370 @@
+        if self.with_overlap:
+            self.main_df["length"] = self.main_df["end"] - self.main_df["start"] + 1
+
+    # pylint: disable=W0612,W0613
+    def profile(
+        self,
+        read_data_provider,
+        restrict_reports=None,
+        report_category=True,
+        report_unannotated=False,
+    ):
+        publish_reports = [
```

Ruff (0.8.0)

Local variable `publish_reports` is assigned to but never used

Remove assignment to unused variable `publish_reports`

(F841)